index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - setup and mobile first</title>
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <div class="toolbar">
+            <div class="container-1">
+                <div class="clock"><p><strong>9:41</strong></p></div>
+                <div class="space2"></div>
+                <div class="conn"><img class="conn-im" alt="Connetion" src="images/connection.png"></div>
+                <div class="wire"><img class="wire-im" alt="Wireless" src="images/wireless.png"></div>
+                <div class="batt"><img class="batt-im" alt="Battery" src="images/battery.png"></div>
+            </div>
+            <div class="container-2">
+                <div class="logo"><a href="#home"><img class="mylogo-im" alt="MyLogo" src="images/mylogo.png"></a></div>
+                <div class="space1"></div>
+                <nav class="menu"><div><img class="menu-im" alt="Menu" src="images/menu.png"></div></nav>
+            </div>
+        </div>
+    </header>
+    <section class="headline">
+        <h1>I'm Juan Carlos. Glad to see you!</h1>
+        <p class="desc">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        <p class="connect">LET'S CONNECT</p>
+        <ul>
+            <div class="container-3">
+                <li>
+                    <a class="twitter" href="www.twitter.com" target="_blank"><img alt="twitter" src="images/twitter.png"></a>
+                </li>
+                <li>
+                    <a class="linkedin" href="www.linkedin.com" target="_blank"><img class="lin" alt="linkedin" src="images/linkedin.png"></a>
+                </li>
+                <li>
+                    <a class="m" href="https://medium.com/" target="_blank"><img alt="logo-m" src="images/m.png"></a>
+                </li>
+                <li>
+                    <a class="github" href="www.github.com" target="_blank"><img alt="github" src="images/github.png"></a>
+                </li>
+                <li>
+                    <a class="sign" href="www.google.com" target="_blank"><img alt="logo-sign" src="images/sign.png"></a>
+                </li>
+            </div>
+        </ul>
+    </section>
```

Comment:
- [ ] As mentioned by the previous reviewer and for the project requirements, kindly make sure your design sitick as much as possible to the Figma design(e.g.: Fonts(size, weigt,...), height, spacing(margin & padding), alignment,...). You can check all details from your chosen Figma template and improve again your design

| Figma Design | Your Design |
| ------------------ | ------------------- |
| ![Screenshotdesign](https://user-images.githubusercontent.com/50990925/227406944-1cc9d20a-2085-44b9-8d71-91382b08b5b3.png) | ![Screenshot from 2023-03-24 18-08-38](https://user-images.githubusercontent.com/83042742/227652762-603b18e7-bf20-4828-8687-f347371abd59.png) |